addons/zylann.scatter/tools/palette.gd

```diff
@@ -103,6 +103,18 @@ func _on_ItemList_multi_selected(_unused_index, _unused_selected):
 func get_configured_margin() -> float:
 	return _margin_spin_box.value
 
+func get_configured_size_range() -> Array:
+	var size_range : Array = [100.0,100.0]
+	var _min_size = get_node("VBoxContainer/SizeContainer/MinSizeValue")
+	var _max_size = get_node("VBoxContainer/SizeContainer2/MaxSizeValue")
```

Comment:
Don't get_node in the middle of the script. Cache in onready on top.

Reply:
Will do.